include/tvm/relay/attrs/reduce.h

```diff
@@ -60,6 +60,37 @@ struct ReduceAttrs : public tvm::AttrsNode<ReduceAttrs> {
         "Whether to perform reduction on axis that are NOT in axis instead.");
   }
 };
+
+struct VarianceAttrs : public tvm::AttrsNode<VarianceAttrs> {
```

Comment:
I wonder if we can inherit from `ReduceAttrs`. Not sure if the attribute system works in that case.

Reply:
If `VarianceAttrs` is inherited from `ReduceAttrs`, `VarianceAttrs` varaibles in python can't visit `unbiased` because it is inherited from `tvm::AttrsNode<ReduceAttrs>` instead of `tvm::AttrsNode<VarianceAttrs>`. For example, `orig.attrs.unbiased` in `_tensor_grad.py:592` will be confronted with the bug.
https://github.com/apache/incubator-tvm/blob/375d377e6134e74ae6b7b489dde6182045b24774/python/tvm/relay/op/_tensor_grad.py#L588-L592

Reply:
ok, thanks